README.md

```diff
@@ -54,52 +42,130 @@ Server to device communication uses firebase push notifications as transport for
+
+```
+curl -X POST \
+  -u {AccountId}:{SecretKey} \
+  https://v3.api.hypertrack.com/devices/{device_id}/start
+```
+
+
+Get the tracking status of the device by calling
+[GET /devices/{device_id}](https://docs.hypertrack.com/?shell#references-apis-devices-get-devices) api.
+
+```
+curl \
+  -u {AccountId}:{SecretKey} \
+  https://v3.api.hypertrack.com/devices/{device_id}
+```
+
+To see the device on a map, open the returned embed_url in your browser (no login required, so you can add embed these views directly to you web app).
+The device will also show up in the device list in the [HyperTrack dashboard](https://dashboard.hypertrack.com/).
 
-#### Step 5. Verify your Integration
+To stop tracking, call the [stop](https://docs.hypertrack.com/?shell#references-apis-devices-post-devices-device_id-stop) API.
 
-Once you've finished pervious steps you are ready to test your integration.
-Simplest way to start tracking is to use [HyperTrack public API](https://docs.hypertrack.com/#references-apis-devices-post-devices-device_id-start) and you need to copy your authentication credentials from [Dashboard](https://dashboard.hypertrack.com/setup), so the start command will look like below:
-```bash
-curl https://v3.api.hypertrack.com/devices/{deviceId}/start \
-    --request POST \
-    --user {AccountId}:{SecretKey} \
 ```
+curl -X POST \
+  -u {AccountId}:{SecretKey} \
+  https://v3.api.hypertrack.com/devices/{device_id}/stop
+```
+
+### Track trips with ETA
+
+If you want to track a device on its way to a destination, call the [Trips API](https://docs.hypertrack.com/#references-apis-trips-post-trips)
+and add destination.
+
+HyperTrack Trips API offers extra fields to get additional intelligence over the Devices API.
+* set destination to track route and ETA
+* set scheduled_at to track delays
+* share live tracking URL of the trip with customers 
+* embed live tracking view of the trip in your ops dashboard 
+
+```curl
+curl -u {AccountId}:{SecretKey} --location --request POST 'https://v3.api.hypertrack.com/trips/' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "device_id": "{device_id}",
+    "destination": {
+        "geometry": {
+            "type": "Point",
+            "coordinates": [{longitude}, {latitude}]
+        }
+    }
+}'
+```
+
+To get `{longitude}` and `{latitude}` of your destination, you can use for example [Google Maps](https://support.google.com/maps/answer/18539?co=GENIE.Platform%3DDesktop&hl=en).
 
-Once the command above is done, you'll see tracking notification on device and will be able to see its location on dashboard.
+> HyperTrack uses [GeoJSON](https://en.wikipedia.org/wiki/GeoJSON). Please make sure you follow the correct ordering of longitude and latitude.
 
-When you're done you can finish the trip using following bash command:
-```bash
-curl https://v3.api.hypertrack.com/devices/{deviceId}/stop \
```

Comment:
You've replaced 
```bash
curl https://v3.api.hypertrack.com/devices/{deviceId}/stop \
    --request POST \
    --user {AccountId}:{SecretKey} \
```
with short form 
```bash
curl https://v3.api.hypertrack.com/devices/{deviceId}/stop \
    -X POST \
    -u {AccountId}:{SecretKey} \
```

Is this intentional? Short switches are good for scripts, but for samples it is better to use full switches as their meaning is clear even for those, who aren't familiar with `curl` syntax.
E.g. windows users might prefer to use PowerShell's [Invoke-RestMethod](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-restmethod?view=powershell-7) command and they have less pain translating it if every argument is clear.
```powershell
$AccountId = "{AccountId}"
$SecretKey = ConvertTo-SecureString -String "{SecretKey}" -AsPlainText -Force
$Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $AccountId, $SecretKey

Invoke-RestMethod 'https://v3.api.hypertrack.com/devices/{deviceId}/stop' -Authentication 'Basic' -Method 'Post' -Credential $Credential
```